### ISSUE TITLE
Custom style for task overlay layer on map

### ISSUE THREAD

**Issue by ebrelsford** [maproulette/maproulette2#216]
It can be hard to see a task on the map, especially depending on which base map you're using. It would be nice to be able to change this challenge-wide (eg, set the stroke color, width, etc).

Here's an example of how hard it can be to see the overlay when using OpenCycleMap:

![selection_001](https://cloud.githubusercontent.com/assets/375002/19737695/7301a288-9b82-11e6-826a-e628d94100c0.png)

**Comment by mvexel** [comment]
Yes, that would be an interesting thing to do. Basically define the color / transparency of the point and line features on the GeoJSON layer.

Would require a (small?) back end change to store these values and additional fields / page in the challenge UI.
